### PR TITLE
Migration Analytics in Production

### PR DESCRIPTION
**Pull request by carlosthe19916**
Migration Analytics will go to Production and I'd like to merge this PR for having our URL into Production.

- Just one question: is it OK to keep `title: Experimental API` [1]? or what should be the correct value?

[1] https://github.com/carlosthe19916/cloud-services-config/blob/patch-5/main.yml#L310

### CODE REVIEW

**Comment by kruai** [comment]
@carlosthe19916 my understanding is that this should become active on 4/23; do you know what time we should aim for?
I'm also not sure what the tags fields are for. Do you know more @karelhala @ryelo ?

**Review by kruai — APPROVED** [review]

**Comment by ryelo** [comment]
Not sure about the experimental API, but 4/23 is the date we have so hold this until then.

File: main.yml
Comment (karelhala) [inline main.yml:306]: This means that in the api-docs page we'll show the app as xavier instead of `Migration Analytics` is that correct?

File: main.yml
Comment (karelhala) [inline main.yml:303]: I tried looking for `migration-analytics` on the `/api` andpoint and it looks like there is no such api, but I could find `/api/xavier` do you mind adding
```yaml
  api:
    ...
    apiName: xavier
    ...
```

File: main.yml
Comment (karelhala) [inline main.yml:309]: Tags currently does not have any effect, but good point in adding these! You can keep these in here, it will just have no effect, do you want to somehow show in the UI that users might experience some problems when accessing this API?

**Review by karelhala — CHANGES_REQUESTED** [review]
Other than `apiName` I am fine with this, few questions but overall looks good!

**Comment by PhilipCattanach** [comment]
@ryelo @kruai 
Hi Asa/Ryan,

This is the first time my Engineering team has deployed a new application to Insights Production so we may need some help/direction from yourselves.

We had hoped to deploy the prod stable branch today. With the intention of allowing the Engineering team and QA team to be able to do some testing to ensure all is well using the URL https://cloud.redhat.com/migrations/migration-analytics.

Then on Thursday (23rd April) the Insights homepage will be updated with the new Migration Services panel with the link to the Migration Analytics URL.

Is that something that can be done? Or do both steps need to be done on the 23rd? 

Thanks,
Phil

**Comment by karelhala** [comment]
@PhilipCattanach it can be merged prior 23rd if you want to test things out. It will be accessible on the URL you posted, but since there is no direct link in he UI to this application I am fine with merging it users won't be able to access it unless they know the URL.

**Comment by kruai** [comment]
I mean, we do have pre-production environments as well... it would be great if we could do QA testing in the QA environment. Why do we need to do QA testing on Production for this app?

**Comment by PhilipCattanach** [comment]
@kruai 
Asa, we have tested the application in QA already. It is that we are being very 'risk adverse' for our first deployment to prod-stable.

File: main.yml
Comment (carlosthe19916) [reply main.yml:306]: We'd like to show `Migration Analytics` instead of `xavier`. I'll add a new commit to change it. Thanks

**Comment by PhilipCattanach** [comment]
@karelhala 
Karel - exactly, you get the point - it will only be the Engineering team and QE who will have access to the application prior 23rd.

File: main.yml
Comment (carlosthe19916) [reply main.yml:303]: @karelhala I'll add it no problem. I just wonder why the other branches worked fine without `apiName: xavier`? I mean, our config for `ci-stable, qa-stable, etc` did not include `apiName: xavier`

File: main.yml
Comment (PhilipCattanach) [reply main.yml:306]: Thanks for the feedback. This PR has the same changes we merged into the qa-stable branch. We will apply the changes that you have suggested starting from ci-stable to validate and promote them.

File: main.yml
Comment (PhilipCattanach) [reply main.yml:303]: As above

File: main.yml
Comment (carlosthe19916) [reply main.yml:309]: let's keep it since it will have no effect... showing messages like `users might experience some problems when accessing this API` were not in our plans so I think it is OK for now not to show such message

Comment (PhilipCattanach) [reply main.yml:309]: This value is fine for this PR. 
Looking ahead - can you provide a list of the other permissible values please so we can decide which tag is most appropriate for Migration Analytics?

**Comment by kruai** [comment]
OK, if nobody broadcasts the URL and it isn't added to the landing page, I'm fine with it! Once Karel's comments have been addressed, I think this will be fine to merge today.

File: main.yml
Comment (karelhala) [reply main.yml:303]: It totaly got out of my radar, currently the xavier API is not listed on API docs because there is missing openAPI spec but with this applied and if you'd were to expose openapi.json on `/api/xavier/v1/openapi.json` customers could check your API and use it directly without using app UI.

File: main.yml
Comment (karelhala) [reply main.yml:306]: Yeah, sorry that I didn't noticed this before, but it totally got out of my radar.

File: main.yml
Comment (karelhala) [reply main.yml:309]: Currently we are not showing any tags in api docs, but there were plans to show them. We were planning to show just the value and title so users can filter the table in docs api.

File: main.yml
Comment (PhilipCattanach) [reply main.yml:306]: @kruai 
Asa - Sorry for the confusion. Myself and Carlos were updating in parallel. Just to reiterate we won't be making a change to this PR. We must release to prod-stable the same configuration that we have tested in ci-stable and qa-stable.

File: main.yml
Comment (PhilipCattanach) [reply main.yml:303]: @kruai 
Asa - Sorry for the confusion. Myself and Carlos were updating in parallel. Just to reiterate we won't be making a change to this PR. We must release to prod-stable the same configuration that we have tested in ci-stable and qa-stable.

**Comment by PhilipCattanach** [comment]
@kruai 
Asa - Are you satisfied with the replies to all of Karel's comments? Can you proceed and apply the PR?

File: main.yml
Comment (karelhala) [reply main.yml:306]: Ah, okay. Sure thing! If you want to discuss the docs api feel free to ping me and we can discuss it more deeply.

**Review by karelhala — APPROVED** [review]

**Comment by kruai** [comment]
Yep, that's fine with me. Merging shortly.

**Pull request merged**
